tests/framework/Samplers/AdaptiveMonteCarlo/test_adaptive_MC_sampler.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <!-- TestInfo -->
+  <TestInfo>
+    <name>framework/Samplers/AdaptiveMonteCarlo</name>
+    <author>ZHOUJ2</author>
+    <created>2020-02-06</created>
+    <classesTested>Samplers.AdaptiveMonteCarlo, Models.ExternalModel</classesTested>
+    <description>
+        This  test is aimed to test the capability of RAVEN to employ sampling with convergence to statistic infomation. It tests the AdaptiveMonteCarlo sampler.
```

Comment:
a bit more description... what are the statistical moments that we test? The model?